pkgs/top-level/all-packages.nix

```diff
@@ -40389,6 +40389,8 @@ with pkgs;
 
   sndio = callPackage ../misc/sndio { };
 
+  sp800-90bEntropyAssessment = callPackage ../tools/misc/sp800-90bEntropyAssessment { };
```

Comment:
What a brutal name :D

Reply:
I know the name is a bit long, but descriptive. I'm open for new naming ideas, but i think just calling it 'entropyAssessment', 'sp80090bEnt' or similar, would not be helping in terms of readability.

Reply:
I mean that's just it's name. There is nothing you can do about it :dagger: 

Because it's so complex, it's probably best if you use the exact upstream name: `SP800-90B_EntropyAssessment`

Reply:
Okay, the commit message and the folder name are now also changed.